ui/components/app/snaps/snap-remove-warning/snap-remove-warning.js

```diff
@@ -6,12 +6,12 @@ import {
   Box,
   Button,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   BUTTON_VARIANT,
   BUTTON_SIZES,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 
 import {
   BlockSize,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 44 21 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0e3d965e-3d42-49e0-a2dd-ce0929665709">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 44 33 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0eeac2e1-2cc3-45f0-9f94-dcd7372fb2cd">